catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -108,27 +112,31 @@ private void transformTargetLocation(Metacard metacard) {
                 WKTReader wktReader = new WKTReader(geometryFactory);
                 Geometry geometry = wktReader.read(locationString);
 
-                LOGGER.debug("Setting the metacard attribute [{}, {}]", Core.LOCATION,
+                LOGGER.debug("Setting the metacard attribute [{}, {}]",
+                        Core.LOCATION,
                         geometry.toText());
                 metacard.setAttribute(new AttributeImpl(Core.LOCATION, geometry.toText()));
             }
 
         } catch (ParseException e) {
-            LOGGER.debug(e.getMessage(), e);
+            LOGGER.warn(e.getMessage(), e);
```

Comment:
it's certainly possible that it was merged/rebased incorrectly. I originally pulled CAL-79, and then based my code from that. Unfortunately, my push to github included the commits messages from CAL-79. I don't know a different way of doing it, but I might just need to rebranch and copy my changes once CAL-79 is committed.